_sass/module/_hero.scss

```diff
@@ -62,10 +63,15 @@ $hero-breakpoint: $grid--medium;
 // Layout element
 .hero__main {
   @include grid-column(12);
+  font-weight: 350;
+  font-style: italic;
+  display: flex;
```

Comment:
I'm not really familiar with flexbox, but the grid is based on floats and margins, so I'm not sure we want to mix them. You probably want to remove the `@include grid-column`, but I'm not sure.